Makefile

```
@@ -74,6 +76,22 @@ down: ## remove stack (warning: it removes the database container)
 	@$(COMPOSE) down || echo WARNING: unable to remove the stack. Try to stop linked containers or networks first.
 .PHONY: down
 
+fixtures: 
+	@$(COMPOSE) up -d elasticsearch
+	@echo "Wait for database to be up..."
+	@$(WAIT_ES)
+	zcat ./fixtures/elasticsearch/lrs.json.gz | \
+	  python3 ./scripts/patch_statements_date.py | \
+	  $(COMPOSE_RUN) ralph \
+		  "bash" \
+		  "-c" \
+		  "ralph push -b es --es-index statements-fixtures"
+	@$(COMPOSE) up -d edx_mysql
+	@echo "Wait for database to be up..."
+	@$(WAIT_MYSQL)
+	scripts/users-permissions.sh
+.PHONY: fixtures
```

As grafana should be running, why not depend on the `run` target?

```suggestion
fixtures: \
    run
fixtures: ## Load test data (for development)
	@echo "Wait for databases to be up..."
	@$(WAIT_ES)
	@$(WAIT_MYSQL)
	zcat ./fixtures/elasticsearch/lrs.json.gz | \
	  $(COMPOSE_RUN) patch_statements_date | \
	  $(COMPOSE_RUN) ralph push -b es --es-index statements-fixtures  && \
	  scripts/users-permissions.sh
.PHONY: fixtures
```

Also the following line implies that you have a dedicated service for this:

```
$(COMPOSE_RUN) patch_statements_date
```